doc/glossary.adoc

```diff
@@ -2,47 +2,58 @@
 
 == LivingDoc 2 - Core Domain
 
-Decision Table:: [[decision-table]] A table where each line represents
-a <<test-case,Test Case>>.  A column can contain either a <<test-input,Test
-Input>> or an <<expectation,Expectation>>.
+Decision Table:: [[decision-table]]
+A table where each row represents a <<test-case,Test Case>>.  A column can
+contain either a <<test-input,Test Input>> or an <<expectation,Expectation>>.
 
-Example:: A description of the behavior of the <<system-under-test,System under
+Example:: [[example]]
+A description of the behavior of the <<system-under-test,System under
 Test>>, which can be automatically verified by LivingDoc. <<scenario,Scenario>>
 and <<decision-table,Decision Table>> are examples.
 
-Executable Document:: [[executable-document]]A document describing the
-<<system-under-test,System under Test>>. It contains zero or more
-<<example,Examples>>.
-
-Expectation:: [[expectation]]A value that is passed to the <<fixture,Fixture>>
-to determine the outcome of a <<test-case,Test Case>>.
-
-Fixture:: [[fixture]]Coded by a developer, a fixture is used by an
-<<interpreter,Interpreter>> to execute a <<test-case,Test Case>>. The fixture
-maps <<test-input,Test Inputs>> to a series of interactions with the
-<<system-under-test,System under Test>> and evaluates
-<<expectations,Expectations>>.
-
-Interpreter:: [[interpreter]] Takes an <<example,Example>>, determines required
```

Comment:
Yes, I agree that _Interpreter_ is too implementation-specific and should be removed from our model for now.